src/pages/PatientManager/PatientSummary.tsx

```diff
@@ -62,56 +83,153 @@ const PatientSummary = ({
+                                  medication === clickedMedication
+                                    ? "bg-indigo-100"
+                                    : ""
+                                }`}
+                                >
+                                  <div className="flex w-0 flex-1 items-center">
+                                    <div className="ml-4 flex min-w-0 flex-1 gap-2">
+                                      <span className="truncate font-medium">
+                                        {medication}
+                                      </span>
+                                    </div>
+                                  </div>
+                                  <div className="ml-4 flex-shrink-0">
+                                    <button
+                                      onClick={() =>
+                                        handleMedicationClick(medication)
+                                      }
+                                      className="font-medium text-indigo-600 hover:text-indigo-500 focus:outline-none"
+                                    >
+                                      Benefits and risks
+                                    </button>
+                                  </div>
+                                </li>
+                              )
+                            )}
+                        </ul>
+                      </dd>
+                    </div>
+
+                    {riskData && (
+                      <div className="px-4 py-6 sm:grid sm:gap-4 sm:px-0">
+                        <dt className="text-sm font-medium leading-6 text-gray-900">
+                          Benefits and risks
+                        </dt>
+                        <dd className="mt-1 text-sm leading-6 text-gray-700 sm:col-span-2 sm:mt-0">
+                          <div className="flex">
+                            <div className="p-2">
+                              <div>
+                                <h4 className="text-sm font-medium">
+                                  Benefits:
+                                </h4>
+                              </div>
+                              <div>
+                                <ul>
+                                  {riskData.benefits.map((benefit: string) => (
+                                    <li className="text-sm">{benefit}</li>
+                                  ))}
+                                </ul>
+                              </div>
+                            </div>
+                            <div className="p-2">
+                              <div>
+                                <h4 className="text-sm font-medium">Risks:</h4>
+                              </div>
+                              <div>
+                                <ul>
+                                  {riskData.risks.map((risk: string) => (
+                                    <li className="text-sm">{risk}</li>
```

Comment:
in each of these maps I think you'll need to give the `li` a `key={index}` to satisfy React.